### ISSUE TITLE
Large page error

### ISSUE THREAD

**Issue by cs-z** [Symphonic3/MonieO#1]
If "Allocate memory in large pages" mining option enabled, the program crashed without an error log. 
RandomX-JNI dlls are successfully initialized anyway.
Windows 10, JRE 1.8.0

**Comment by Symphonic3** [comment]
Could you run the program from a terminal window and paste the full log? What are your system specs? (CPU, RAM)

**Comment by Symphonic3** [comment]
Allocate memory in large pages should be enabled by default and the test hash on startup should be using it. Does mining work if you leave settings as default and begin mining?

**Comment by cs-z** [comment]
Yes, it does work, and the test hash is correct. The confusing thing was that the long page checkbox isn't set by default, and when i set it manually, it crashes when starting mining.
